README.md

```diff
@@ -183,3 +183,24 @@ The PR is the only place any team member is expected to consult for information
 [soliumrc]: ./.soliumrc.json
 [travisci]: https://travis-ci.org/
 [webpack]: https://webpack.js.org/
+
+
+### Managing internal and external dependencies
```

Comment:
Almost done 🎢 
Please move the entire section at the end of the `package.json` section.